CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
Do we plan to do this across all the language SDKs? Should we copy them verbatim?
https://github.com/Azure/azure-sdk-for-java/blob/master/CODE_OF_CONDUCT.md
https://github.com/Azure/azure-sdk-for-python/blob/master/CODE_OF_CONDUCT.md

.NET doesn't seem to have it as a separate doc, but rather part of the contributing section:
https://github.com/Azure/azure-sdk-for-net#contributing

Reply:
I took the Code of Conduct that OSS requires us to have in the repo, https://github.com/microsoft/repo-templates/blob/main/shared/CODE_OF_CONDUCT.md